wordpress-mobile/WordPress-Android
# Use View Binding in Reader Activities and Fragments

This PR introduces View Bindings for the Reader Activities and Fragments. It initializes and uses the `binding` object in a pretty standard way, following the best practices described [here](https://developer.android.com/topic/libraries/view-binding), but I'm not sure I like how it looks like for Fragments and I'm open to other suggestions.

Note: I've added a `@Suppress("ComplexMethod")` to the method that handles navigation events, as I can't think of a way to simplify it any further. Let me know if you any suggestions.

To test:

Smoke test the Reader, with special attention for the Discover section, the Interests screen and the Post Detail screen. Make sure everything is working as expected and nothing looks out of place. Also try to reblog a post with an account with no sites and make sure everything looks fine on the No Site To Reblog screen.

PR submission checklist:
